babel.config.json

```diff
@@ -1,19 +1,22 @@
 {
-  "presets": [
-    [
-		"@babel/preset-env", {
-			"loose": true
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"loose": true
+			}
+		]
+	],
+	"env": {
+		"umd": {
```

Comment:
Do we still need `umd` here?

The reformatting could probably be broken off as a separate PR

Reply:
Oh, we don't. Removed.

Reply:
> The reformatting could probably be broken off as a separate PR

Did you mean `babel.config.json` reformatting? That one was done by some auto formatter, I was just too lazy to tidy it up.

Reply:
Yeah, that's what I referring to. I figured it could either be its own PR or dropped to make it clearer that nothing's changing there

Reply:
Removed that change